aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -129,14 +129,17 @@ const auto ndtri_string = jiterator_stringify(
   * coef[0] = C  , ..., coef[N] = C  .
   *            N                   0
   */
-template <typename T>
-T polevl(const T x, const T A[], const int len) {
-  T result = 0;
-  for (int i = 0; i < len; ++i) {
-    result = result * x + A[i];
+  template <typename T>
+  T polevl(const T x, const T A[], const int len) {
+    // NOTE: This `polevl` is different from other `polevl`
+    // implementation (in PyTorch) which expect the `len`  to be
```

Comment:
This comment reads incorrectly. This version of polevl should be passed the size of an array. The other version should be passed the size + 1 because it's the weird one.

Reply:
(I can fix this in a follow-up PR)

Reply:
Updated.